privacyidea/lib/eventhandler/customuserattributeshandler.py

```diff
@@ -0,0 +1,110 @@
+__doc__ = """This is the event handler module for custom user attributes.
+You can set or delete custom your attributes
+
+"""
+
+from privacyidea.lib.eventhandler.base import BaseEventHandler
+from privacyidea.lib.user import User
+from privacyidea.lib import _
+import logging
+
+log = logging.getLogger(__name__)
+
+class USER_TYPE(object):
+    """
+    Allowed token owner
+    """
+    TOKENOWNER = "tokenowner"
+    LOGGED_IN_USER = "logged_in_user"
+
+
+class CustomUserAttributesHandler(BaseEventHandler):
+    """
+    An Eventhandler needs to return a list of actions, which it can handle.
+
+    It also returns a list of allowed action and conditions
+
+    It returns an identifier, which can be used in the eventhandlig definitions
+    """
+
+    identifier = "CustomUserAttributes"
+    description = "This event handler can set and delete custom_user_attributes"
+
+    @property
+    def allowed_positions(cls):
+        """
+        This returns the allowed positions of the event handler definition.
+        :return: list of allowed positions
+        """
+        return ["post", "pre"]
+
+    @property
+    def actions(cls):
+        """
+        This method returns a dictionary of allowed actions and possible
+        options in this handler module.
+
+        :return: dict with actions
+        """
+        actions = {
+            "set_custom_user_attributes": {
```

Comment:
We should create constants for the actions like here:
https://github.com/privacyidea/privacyidea/blob/7bbf194aebfd2bbbee9b610ab22763ad5e6b32cb/privacyidea/lib/eventhandler/tokenhandler.py#L66